model/src/payout_curve/quanto.rs

```diff
@@ -621,6 +621,7 @@ mod api_tests {
 #[cfg(test)]
 mod unit_tests {
     use super::*;
+    use crate::payout_curve::ETHUSD_MULTIPLIER;
```

Comment:
🔧 I introduced an arbitrary multiplier just below. Shouldn't need to import the `ETHUSD` one.

Reply:
I think it shouldn't matter what constant you use to check for regressions!